docs/rum_collection/_index.md

```diff
@@ -49,30 +49,40 @@ For more information about setting up a client token, see the [Client token docu
 
 ### Initialize the library
 
+{{< site-region region="us" >}}
 {{< tabs >}}
-{{% tab "US" %}}
+{{% tab "Swift" %}}
 
 ```swift
 Datadog.initialize(
     appContext: .init(),
     trackingConsent: trackingConsent,
     configuration: Datadog.Configuration
-        .builderUsing(
-            rumApplicationID: "<rum_application_id>",
-            clientToken: "<client_token>",
-            environment: "<environment_name>"
-        )
+        .builderUsing(clientToken: "<client_token>", environment: "<environment_name>")
```

Comment:
did you remove `rumAppID` by mistake? or purposefully?
i guess it should be used here

Reply:
Copy/paste mistake!